docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md

```diff
@@ -24,39 +24,50 @@ YugabyteDB Anywhere allows you to expand a universe to add more nodes and shrink
 
 ## Edit a universe
 
-To change the configuration of a universe, navigate to **Universes > UniverseName > Actions > Edit Universe**, as per the following illustration:
+To change the configuration of a universe, do the following:
 
-![Edit universe](/images/ee/edit-univ-220.png)
+1. Navigate to your universe and choose **Actions > Edit Universe** to display the **Edit universe** page.
 
-Using the **Edit Universe** page, you can modify the following:
+    ![Edit universe](/images/ee/edit-univ-220.png)
 
-- Regions - you can select any region configured in the provider used to deploy the universe
-- Number of nodes and Availability zones - as you add nodes, they are automatically distributed among the availability zones; you can also add, configure, and remove availability zones
-- [Master placement](../../create-deployments/dedicated-master/)
-- Instance type and volume size - you can select instance types configured in the provider
-- [User tags](../instance-tags/)
+1. Update the configuration.
 
-YugabyteDB Anywhere performs these modifications through the [YB-Masters](../../../architecture/yb-master/) powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
+    Using the **Edit Universe** page, you can modify the following:
 
-Note that you can't change the replication factor of a universe.
+    - **Regions** - you can select any region configured in the provider used to deploy the universe.
+    - [Master Placement](../../create-deployments/dedicated-master/).
+    - **Total Nodes** and **Availability Zones** - as you add nodes, they are automatically distributed among the availability zones; you can also add, configure, and remove availability zones.
+    - **Instance Configuration** - change instance type and volume size as configured in the provider. In some cases, these operations are available as a [smart resize](#smart-resize).
+    - [User tags](../instance-tags/).
```

Comment:
Technically, changing just the user tags for a cloud provider also is in-place (doesn't require touching the YBDB processes as it is a cloud provider option). Don't think we want to call it a "smart resize" but maybe we could say something like 

- User tags (changing just the tags for a cloud provider based universe does not involve any node restarts or data moves)

But we could also ignore this case.